db/migrate/20220104111953_add_timestamps_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddTimestampsToArticles < ActiveRecord::Migration[6.1]
```

Comment:
Why separate migration? why not put it in the same migration?

Reply:
it was a separated thing in the video, now changing